doc/articles/features/app-links.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: Uno.Features.AppLinks
+---
+
+# App Links
+
+> [!TIP]
+> This article covers a quick overview for enabling Universal/App Links. For a full description of the feature and instructions on using it in general, see the [Microsoft docs on Universal Links](https://learn.microsoft.com/dotnet/maui/macios/universal-links) and [App Links](https://learn.microsoft.com/dotnet/maui/android/app-links).
+
+* Universal Links (iOS) and App Links (Android) are mechanisms that allow deep linking to specific content within your application directly from external sources, such as websites or other apps.
+
+## Using Universal/App Links with Uno
+
+### iOS Universal Links
+
+1. **Update the `Main.iOS.cs` file:**
+   * Add `OpenUrl` method override in your `Main` to handle Universal Links:
+
+   ```csharp
+   // ...
+
+   public class App : UIApplication
+    {
+        public override void OpenUrl(NSUrl url, NSDictionary options, Action<bool>? completion)
```

Comment:
We're supposed to support this with `OnActivated` arguments, did it now work when you tried?